docs/tutorials/trig.rst

```diff
@@ -0,0 +1,89 @@
+.. _trig_approx_tutorial:
+
+*****************************************
+Fast trigonometric altitude approximation
+*****************************************
+
+Contents
+========
+
+* :ref:`trig-background`
+* :ref:`trig-trig_approx`
+
+.. _trig-background:
+
+Background
+==========
+
+astroplan ordinarily computes the rise and set times of an object by
+transforming the sky coordinate of the object (e.g.~ICRS, galactic, etc.) into a
+grid of altitude-azimuth coordinates for that target as seen by an observer at a
+specific location on the Earth, at 10 minute intervals over a 24 hour period.
+The rise or set time is then computed by linear interpolation between the two
+coordinates nearest to zero. The meridian/anti-meridian transit time is computed
+similarly; it takes a numerical derivative of the altitudes before searching for
+the appropriate zero crossing.
+
+We chose to compute rise and set times with a grid-search to maximize accuracy,
+rather than speed. In particular, we sought to preserve the astropy
+altitude-azimuth coordinate transformation which accounts for atmospheric
+refraction. If you have a need for speed, however, there's an alternative...
+
+.. _trig-trig_approx:
+
+The ``trig_approx`` option
+==========================
+
+A simple trigonometric algorithm is included within the `~astroplan.Observer`
+object for computing rise and set times without accounting for refraction, which
+yields a 3x speed gain in exchange for diminished accuracy. The fast,
```

Comment:
I would suggest caveating this with something like "yields a speed gain (3x as of Astropy v2.0 and astroplan vx.y)" in case there are changes in performance later on.